_data/simple-icons.json

```diff
@@ -7327,8 +7327,8 @@
         },
         {
             "title": "Zulip",
-            "hex": "52C2AF",
-            "source": "https://github.com/zulip/zulip/"
+            "hex": "FFFFFF",
+            "source": "https://github.com/zulip/zulip/tree/master/static/images/logo"
```

Comment:
Is it worth updating the source to the website? They now include the SVG logo on the homepage.

Reply:
I think using GitHub as a source is better, it'll probably be easier to find an updated version of the logo there if we need it.

However, as per [the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), this URL should include the commit hash of the commit in which the SVG that we used was added.